web/src/compose_validate.ts

```diff
@@ -712,13 +715,12 @@ export function check_overflow_text(): number {
             $("#compose_banners"),
         );
         set_message_too_long(true);
-    } else if (text.length > 0.9 * max_length) {
+    } else if (text.length >= 0.91 * max_length) {
```

Comment:
I guess is this intended to avoid having it be higher than 900?

Reply:
That's right

Reply:
But wouldn't the original logic still produce 999 and still not wrap?

I think the right way to write this is to just check if the characters remaining is at most 999, that'll work even if we change the numbers.

Reply:
The number was 999 instead of 900... have fixed it and pushed now!